scripts/release/generate-changelog.js

```diff
@@ -12,6 +12,8 @@ const { modifyFile } = require('../lib/files-utils')
 
 const CHANGELOG_FILE = 'CHANGELOG.md'
 const CONTRIBUTING_FILE = 'CONTRIBUTING.md'
+const EMOJI_PRIORITY = ['💥', '✨', '🐛', '⚗️', '♻️']
+const START_WITH_EMOJI = new RegExp(/^(\p{Emoji_Presentation})/u)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
const START_WITH_EMOJI = /^\p{Emoji_Presentation}/u
```